scripts/mafia.js

```diff
@@ -3287,23 +3287,23 @@ function Mafia(mafiachan) {
             if (p === name) {
                 continue;
             }
-            if (!(mafia.players[p].role.hasOwnProperty("voteHax"))) {
+            if (!mafia.players[p].role.actions.hasOwnProperty("voteHax")) {
                 continue;
             }
-            var voteHaxData = mafia.players[p].role.voteHax,
+            var voteHaxData = mafia.players[p].role.actions.voteHax,
                 haxmsg = voteHaxData.msg ? voteHaxData.msg : "~Player~ voted for ~Target~!",
                 haxperc = voteHaxData.chance ? voteHaxData.chance : 1;
                 var avoid = player.role.actions.avoidVoteHax;
-                if (Array.isArray(avoid)) {
-                    if (avoid.indexOf(mafia.players[p].role) !== -1) {
-                        avoid = true;
-                    }
-                    else {
-                        avoid = false;
-                    }
+            if (Array.isArray(avoid)) {
+                if (avoid.indexOf(mafia.players[p].role.role) !== -1) {
```

Comment:
`avoid = avoid.indexOf(mafia.players[p].role.role) !== -1` is shorter

Reply:
True, that works. I didn't want to change what the person before me did, but I'll change this the next time I submit something